src/common/src/hash/consistent_hash/vnode.rs

```diff
@@ -87,6 +87,14 @@ impl VirtualNode {
         Self(scalar as _)
     }
 
+    pub const fn vnode_data_type() -> DataType {
```

Comment:
It appears that `RW_TYPE` already exists.

Reply:
fixed.